src/tools.json

```diff
@@ -33,28 +33,28 @@
         "vendor": "Red Hat, Inc.",
         "name": "oc",
         "cmdFileName": "oc",
-        "version": "3.11.0",
-        "versionRange": "^3.11.0",
-        "versionRangeLabel": "version >= 3.11.0 and < 4.0.0",
+        "version": "4.2.2",
+        "versionRange": "^4.2.2",
+        "versionRangeLabel": "version >= 3.11.0 and <= 4.2.2",
```

Comment:
^4.2.2 version range means `version >= 3.11.0 and < 5.0.0` according to semver module doc https://www.npmjs.com/package/semver#caret-ranges-123-025-004.
We should allow to use oc found in path if it differs only in patch version ~4.2.2 which means `version >= 4.2.2  and < 4.3.0`.